packages/data-stores/src/plans/test/selectors.ts

```diff
@@ -0,0 +1,326 @@
+/**
+ * External dependencies
+ */
+import * as deprecate from '@wordpress/deprecated';
+
+/**
+ * Internal dependencies
+ */
+import * as Selectors from '../selectors';
+import * as MockData from '../mock';
+import { buildPlanFeaturesDict } from '../test-utils';
+import { TIMELESS_PLAN_ECOMMERCE, TIMELESS_PLAN_FREE, FREE_PLAN_PRODUCT_ID } from '../constants';
+import type { State } from '../reducer';
+
+// Test data
```

Comment:
I think it makes more sense to move all the mocks to the `/mock/` folder too? What do you think?

Reply:
The `mock/` folder contains sample data, while the `mock` variables at the top of this file are simulating different combinations of feature/plans and locales.

They are just recreating the reducer `state` by using mock data, similarly to what we [already do in many other tests](https://github.com/Automattic/wp-calypso/pull/50492/files#diff-705af06b7e1aeb4a5996de8d769e24dad67a1d205e160026e4bcba8b0faf8e28R26-R35). I almost see them as a test implementation detail, not mock data.